### DISCUSSION TITLE
Protect the source model?

### DISCUSSION THREAD

**Discussion by gerritkk** [google/model-viewer#2759] [Q&A]
Hey I've been trying for a while now to protect the .glb/.gltf file in the src attribute from downloading. Can't block people from downloading it with .htaccess cuz then the model viewer won't load the source file. Is there a solution for this?

For example:
modelviewer.dev/shared-assets/models/glTF-Sample-Models/2.0/DamagedHelmet/glTF/DamagedHelmet.gltf

Is it possible to prevent people from downloading the source model when they go to the source link but let <model-viewer> still load and show the model?

**Comment by elalish** [comment]
No, it's not. Think of MV as just like an `<img>` tag. The only way to protect your 3D content is to make it available at lower resolution than you plan to sell, or to watermark it, or to go through legal channels for copyright infringement.
